game/src/challenges/mod.rs

```diff
@@ -144,12 +143,13 @@ impl ChallengesPicker {
             .draw(ctx)
             .centered_horiz(),
             ctx.style()
-                .btn_outline_text("Introduction and tutorial")
+                .btn_outline
+                .text("Introduction and tutorial")
                 .build_def(ctx)
                 .centered_horiz()
                 .bg(app.cs.panel_bg)
                 .padding(16)
-                .outline(ctx.style().btn_solid.outline),
+                .outline(ctx.style().btn_tab.outline),
```

Comment:
I renamed btn_solid to btn_tab to indicate where it should be used. There were just a few places where it was being used outside of tabs, which I've also addressed in this PR.